Assets/MRTK/Tools/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -210,6 +236,19 @@ private CSProjectInfo GetProjectInfo(Dictionary<string, CSProjectInfo> projectsM
                 toReturn.AddDependency(GetProjectInfo(projectsMap, asmDefInfoMap, builtInPackagesWithoutSource, reference, projectOutputPath));
             }
 
+            // Manually add special plugin dependencies to the projects
+#if UNITY_2019_3_OR_NEWER
+            if (toReturn.Name.StartsWith("Microsoft.MixedReality.Toolkit"))
+            {
+                foreach (var plugin in SpecialPluginNameMappingUnity2019.Values)
+                {
+                    if (projectsMap.TryGetValue(plugin, out CSProjectInfo projectInfo))
+                    {
+                        toReturn.AddDependency(projectInfo);
```

Comment:
Will this add both "Unity.ugui" and "UnityEngine.UI" to the list, or is the original name caught earlier and not added?

Reply:
Hmm... I am not sure what the answer is but perhaps we don't have to worry as long as it works?

Reply:
My guess would be there is only one valid reference (i.e. there could be a fake reference to something but the code here is adding the "real" reference).

Reply:
Might still be worth cleaning up the "fake" one from this list, but if it works for now it's probably fine

Reply:
Just confirmed that there is not any fake entry for ugui on this list.